_config.yml

```diff
@@ -136,3 +137,12 @@ defaults:
       ehc_version: "3.4"
```

Comment:
Should be 3.5

Reply:
I suspect `layout: "docs33_page"` is incorrect.

Reply:
Thats what 3.3 and 3.4 used... I assumed that 3.5 would be the same.

Reply:
This section governs the defaults for the pages under: `documentation/3.4/`.  I'm pretty confident that this is correct as it stands.